scripts/set-versions.sh

```diff
@@ -28,31 +29,51 @@ log_and_exit () {
 }
 
 perform_updates () {
-  echo -e "creating release\nsemver version: $SEMVER_VERSION\nversion number: $VERSION_NUMBER"
-
   # update package.json
-  tmp="package.json_temp"
-  jq --arg semverVersion "$SEMVER_VERSION" '.version = $semverVersion' package.json > "$tmp"
-  mv "$tmp" package.json
+  tmp="${PACKAGE_JSON_FILE}_temp"
+  jq --arg semverVersion "$SEMVER_VERSION" '.version = $semverVersion' $PACKAGE_JSON_FILE > "$tmp"
+  mv "$tmp" $PACKAGE_JSON_FILE
```

Comment:
```suggestion
  jq --arg semverVersion "$SEMVER_VERSION" '.version = $semverVersion' $PACKAGE_JSON_FILE | sponge $PACKAGE_JSON_FILE
```

Do we have [`sponge`](https://linux.die.net/man/1/sponge) ([`moreutils`](https://launchpad.net/ubuntu/trusty/+package/moreutils)) available?

Reply:
I don't know... will check